app/qml/main.qml

```diff
@@ -28,7 +28,7 @@ ApplicationWindow {
     width:  __appwindowwidth
     height: __appwindowheight
     visibility: __appwindowvisibility
-    title: qsTr("Input")
```

Comment:
@PeterPetrik - is this a valid way of making untraslatable?

Reply:
I would just remove qsTr and make a comment "/* do not translate, not visible in app */